warehouse/oidc/views.py

```diff
@@ -340,7 +340,7 @@ def mint_token(
         metrics = request.find_service(IMetricsService, context=None)
         metrics.increment("warehouse.oidc.mint_token.github_reusable_workflow")
 
-    return {"success": True, "token": serialized}
+    return {"success": True, "token": serialized, "expires": expires_at}
```

Comment:
Need to update around L42 to include this in the TypedDict definition

Reply:
Fixed!